utils/rpms/cart.spec

```diff
@@ -137,6 +139,12 @@ ln %{?buildroot}%{carthome}/{TESTING/.build_vars,.build_vars-Linux}.sh
 %{carthome}/.build_vars-Linux.sh
 
 %changelog
+* Mon Nov 11 2019 Brian J. Murrell <brian.murrell@intel.com> - 1.6.0-4
+- Don't R: ompi-devel from cart-devel as it breaks the ior
```

Comment:
not sure which Don't R: is supposed to mean here

Reply:
R: is short for Requires: in the RPM building circles.